app/javascript/multi-file-upload.js

```diff
@@ -11,9 +11,9 @@ MOJFrontend.MultiFileUpload.prototype.uploadFiles = async function(files) {
 
   const uploads = []
 
-  for(var i = 0; i < files.length; i++) {
+  for(const file of files) {
     uploads.push(
-      this.uploadFile(files[i]).then(
+      this.uploadFile(file).then(
         function(response) { console.log(response); }
```

Comment:
Should these console logs be removed?

Reply:
Removed yeah and we need to actually handle whatever comes back error-wise

Reply:
I'll remove the console logs for security purposes - but the actual handling of the error seems like a separate tech debt ticket (this is code we already had in the system, the only reason the PR says I added it is because the file name was changed)